Source/WebService/Blackjack.WebService/ClientApp/src/app/types/dealer.ts

```diff
@@ -0,0 +1,42 @@
+import { Gamer } from "./gamer";
+
+export class Dealer extends Gamer {
+    /**
+     * Determines if the dealer is playing and whether his cards are revealed 
+     */
+    public dealersTurn: boolean = false;
```

Comment:
```suggestion
    public isDealerTurn: boolean = false;
```

Naming bool flags a common practice is to preface the var's name with the word, "is" isActive, isEnabled, etc.

Also typical to name the variable in the affirmative. i.e. you would go with isActive, the affirmative, not, isInactive, the negative.

Also try to avoid using possessive verbiage in variable names as in some scenarios it can be hard to distinguish between plurality and possessiveness. That's not an issue here but can be a problem when making variables possessive in name without the ability to use an apostrophe to denote possession.